package.json

```diff
@@ -58,12 +68,11 @@
     "prebench": "npm run --silent build:cjs",
     "bench": "node scripts/benchRunner",
     "bookmarklet": "node scripts/bookmarklet",
-    "build:es": "cross-env BABEL_ENV=es babel source --out-dir es",
+    "build:es": "cross-env BABEL_ENV=es babel source --out-dir es && node ./scripts/addModulePackageScope.js",
```

Comment:
the type of interpretation is set based on the nearest `package.json` file, so we can just add a `package.json` with `{"type": "module"}` to a directory to make files in that directory being treated as modules even if the root is treated as commonjs (and still use `.js` for all files!)